google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -1219,6 +1221,35 @@ public void testGetDatasetWithSelectedFields() {
     assertNull(dataset.getMaxTimeTravelHours());
   }
 
+  @Test
+  public void testGetDatasetWithAccessPolicyVersion() {
+    String accessPolicyDataset = RemoteBigQueryHelper.generateDatasetName();
+    User user = new User("liamhuffman@google.com");
```

Comment:
Let's use a managed identity for this kind of thing rather than your own identity, particular for an IT test.  Same for the other tests that use your identity directly.

Reply:
I've updated the user to use a default bigquery account for now. I found the account in my test project and I'm not sure if every test account follows the same naming pattern. We discussed a bit offline but this works as a workaround for the time being until we figure out how to integrate a managed account.

Reply:
It appears that not all projects have the test account, as the integration tests failed due to the account not existing in the project. Looking into https://g3doc.corp.google.com/company/teams/eip-cloud/overground/user-guide/policies/org/iam.allowedPolicyMemberDomains.md?cl=head

Reply:
Looks like using the default credentials works in this test project, it was just failing in my personal test project. The tests are using the default managed identity now.